unime/src/routes/badges/[id]/+page.svelte

```diff
@@ -49,29 +49,18 @@
     isFavorite = credential.metadata.is_favorite;
     title = credential.metadata.display.name || credential.data.type.at(-1);
     icon = credential.metadata.display.icon || 'User';
-    // color =
-    //   credential.metadata.display.color ||
-    //   colors.at(
-    //     credential.id
-    //       .match(/[0-9]+/)
-    //       .at(0)
-    //       .at(0) % 8, // TODO: omits last value (white)
-    //   );
   }
 
-  // create entries to be shown
-  const { enrichment, ...entries } = credential.data.credentialSubject.achievement;
-  // entries['issuer'] = credential.data.issuer ?? credential.issuer_name;
-  // entries['issuanceDate'] = new Date(credential.data.issuanceDate).toLocaleString('en-US', {
-  //   dateStyle: 'long',
-  //   timeStyle: 'medium'
-  // });
+  const hiddenStandardFields: string[] = ['id', 'type', 'name', 'description', 'image'];
+  const hiddenCustomFields: string[] = ['enrichment'];
```

Comment:
Feels like something that eventually should be solved in the backend. In any case it could use a comment to add context, e.g:
```typescript
// Custom fields to hide related to NGDIL demo.
const hiddenCustomFields: string[] = ['enrichment'];
```

Reply:
I think `hiddenCustomFields` is a good name for what it is, but adding a comment would me nice :+1: 

Maybe you're right that indeed what to hide and what not is too subjective to even make suggestions about :+1:

Reply:
We could make suggestions about it, but probably in a larger comment explaining why we feel those fields are not necessary to be displayed.